curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1175,33 +1282,43 @@ CURVEFS_ERROR DataCache::Flush(Inode* inode, bool force) {
                     objectName, data_ + writeOffset, n);
```

Comment:
Is there data_ should be data?   we should not memcpy before upload,  the memcpy will be time consuming

Reply:
fixed